bin/push-docker-tags.sh

```diff
@@ -59,9 +59,13 @@ pushTag () {
   fi
 }
 
-if [[ $GIT_TAG =~ ^v[0-9]+ ]]; then
+if [[ $GIT_TAG =~ ^v[0-9]+\.[0-9]+\.[0-9]+-rc ]]; then
+  pushTag "$GIT_TAG"
+
+elif [[ $GIT_TAG =~ ^v[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
   pushTag "$GIT_TAG"
   pushTag "latest"
```

Comment:
Consider dropping the `latest` tag.  It may be confusing whether it refers to the latest release, to an RC, or something else.

Reply:
I agree in principle, but in practice we get issues when tags that have been around a while stop getting updates, so it's probably least surprising to keep it working. That's also why I have begrudgingly restored the release tag here too.